src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/SplitNCigarReads.java

```diff
@@ -119,7 +126,7 @@ public void onTraversalStart() {
 
     @Override
     public void apply(GATKRead read, ReferenceContext referenceContext, FeatureContext featureContext) {
-        splitNCigarRead(rnaReadTransform.apply(read),overhangManager);
+        splitNCigarRead(read, overhangManager);
```

Comment:
Argh, so it seems like `NDNCigarReadTransformer` is another one that wants to go after read filtering rather than before, since it blows up given an invalid cigar (which would be caught by the default read filters). So out of our three current read filters, two (`NDNCigarReadTransformer` and `BQSRReadTransformer`) want to be applied after filtering, and one (`FixMisencodedBaseQualityReads`) wants to go before filtering. @lbergelson @cmnbroad can you comment on this issue?